lib/rubocop/target_finder.rb

```diff
@@ -175,7 +175,7 @@ def configured_include?(file)
     end
 
     def ruby_executable?(file)
-      return false unless File.extname(file).empty? && File.exist?(file)
+      return false unless File.extname(file).empty? && File.exist?(file) && !File.empty?(file)
```

Comment:
The added `!` conditions in the `unless` clause seem to have made it harder to read. How about replacing it with an `if` instead?

```suggestion
      return false if !File.extname(file).empty? || !File.exist?(file) || File.empty?(file)
```

Reply:
Looks good, fixed